benchmark_runner/common/oc/oc.py

```diff
@@ -1387,7 +1387,7 @@ def generate_odf_must_gather(self, destination_path: str = '/tmp', odf_version:
         :raises: RuntimeError if the command fails.
         """
         if not odf_version:
-            raise ValueError("ODF version must be provided")
+            odf_version = ".".join(self.get_odf_version().split(".")[:2])
```

Comment:
If get_odf_version() fails, I presume it raises an exception?  Also, what if it returns something, but that isn't valid?

Reply:
fixed